implementations.md

```diff
@@ -0,0 +1,11 @@
+# Implementations of Container Specification
+
+## Runtime
+
+* [opencontainers/runc](https://github.com/opencontainers/runc) - reference implementation of OCI runtime
+* [hyperhq/runv](https://github.com/hyperhq/runv) - Hypervisor-based runtime for OCI
+
+## Testing
+
+* [huawei-openlab/ocp-testing](https://github.com/huawei-openlab/ocp-testing) - Testing framework for OCI configuration and runtime
```

Comment:
Looking through the code I don't think that the ocp-testing implements what we need for testing the open container spec and we shouldn't point people at it for now. For example see this [source file](https://github.com/huawei-openlab/ocp-testing/blob/master/Cases/Benchmark/monitor/cpumonitor/source/cpu_usage_breakdown.go)

Things I would like to see on an OCI testing tool:

- Validate the configuration of a bundle
- Generate and test a bundle/image cryptographic identity
- Generates an OCI bundle that tests that the runtime is compliant
- A tool for testing fetching/unpacking of OCI images

And these things are things we should probably add to this repo as unit tests for the specification.

Right now this seems to do a lot of testing of docker, which is great, but it isn't directly related to what we are doing with the spec at the moment and is confusing.